creator-node/src/config.js

```diff
@@ -501,6 +438,69 @@ const config = convict({
     env: 'enableRehydrate',
     default: true
   },
+
+  /** sync / snapback configs */
+
+  debounceTime: {
```

Comment:
if we're refactoring these we may be able to nest these into an object. not sure how difficult it would be to represent in env vars https://github.com/mozilla/node-convict/blob/master/packages/convict/test/validation.test.js#L19

Reply:
ye i just wanted to re-order stuff to group all the sync keys together bc i found i couldn't remember which keys there were since we have so many now

would be nice to nest these into an object but don't think that makes sense to take on as part of this PR